crates/subspace-farmer/src/single_disk_farm/piece_cache.rs

```diff
@@ -1,22 +1,28 @@
 #[cfg(test)]
 mod tests;
 
+#[cfg(windows)]
+use crate::single_disk_farm::unbuffered_io_file_windows::UnbufferedIoFileWindows;
+use crate::single_disk_farm::unbuffered_io_file_windows::DISK_SECTOR_SIZE;
```

Comment:
Shouldn't we put it behind `#[cfg(windows)]` as well?

Reply:
No, `DISK_SECTOR_SIZE` is used on all platforms to align the file size to physical disk sector size. This is because we may end up doing direct I/O on other platforms in the future, but there are some platform-specific considerations. For example on Linux if file is open more than once and one of the opening is not direct I/O, file will not be operating in direct I/O mode, though this should be less of a concern with this PR now that everything has migrated over.